packages/swingset-runner/demo/encouragementBotComms/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+import { E } from '@agoric/eventual-send';
```

Comment:
it might be time to make swingset-runner import these directly from SwingSet/demo, rather than copying them over here.. I suspect this will be prone to bitrot

Reply:
Although derived from their SwingSet/demo counterparts, these are not perfect copies.  Several changes were necessary to work in the different environment.